src/main/scala/com/evolutiongaming/cluster/ActorRefExtension.scala

```diff
@@ -0,0 +1,5 @@
+package com.evolutiongaming.cluster
+
+import akka.actor.{ActorRef, Extension}
+
+class ActorRefExtension(val ref: ActorRef) extends Extension
```

Comment:
Sorry, I still don't understand you in full. Do you suggest to create a base trait for `MappedAllocationStrategyDDProxy` and `AdaptiveAllocationStrategyDDProxy`?